src/components/calculator.js

```diff
@@ -0,0 +1,99 @@
+import React, { useState } from 'react';
+import './calculator.css';
+
+const Calculator = () => {
+  const initialDisplay = {
+    total: '',
+    next: null,
+    operation: null,
+  };
+  const [result, setResult] = useState(initialDisplay);
+
+  const calculate = () => {
+  };
+
+  const handleClick = () => {
+    // const buttonName = e.target.textContent;
+    // setResult(calculate(buttonName));
+    setResult(calculate());
+  };
+
+  return (
+    <div className="container">
+      {/* <Button data={result.next || result.total || '0'} /> */}
```

Comment:
- [ ] It's considered good practice to clear your code of commented(unused) code, so please remove all commented code here.